assets/src/application/ui/input/Button/ButtonGroup/style.scss

```diff
@@ -43,15 +47,15 @@
 
 	+ .ee-btn {
 		// TODO: rethink this approach such that btn group margins shouldn't rely on nearby buttons
-		margin-left: var(--ee-margin-default);
+		// margin-left: var(--ee-margin-default);
 
-		[dir='rtl'] & {
-			margin-left: 0;
-			margin-right: var(--ee-margin-default);
-		}
+		// [dir='rtl'] & {
+		// 	margin-left: 0;
+		// 	margin-right: var(--ee-margin-default);
+		// }
 
-		@include max782px {
-			margin-left: 0;
-		}
+		// @include max782px {
+		// 	margin-left: 0;
+		// }
```

Comment:
:+1:

Reply:
oops... meant to remove those comments...